src/main/fc/init.c

```diff
@@ -210,6 +210,8 @@ static IO_t busSwitchResetPin        = IO_NONE;
 #endif
 
 
+bool debugResetEEPROM = false; // To reset EEPROM under debugging session, without power cycling.
```

Comment:
This seems to be incomplete, and will be optimised out in every possible case.

Reply:
It's referenced in `init` after `buttonsInit`, and becomes active `USE_BUTTONS`, `BUTTON_A_PIN` and `BUTTON_B_PIN` are all defined.

Well, the declaration itself needs a conditional, though...

Reply:
Ok, I still do not get how this will make a change in how the firmware works. `debugResetEEPROM` is a constant that evaluates to false, and will be optimised out. How will this change the behaviour?

Reply:
`debugResetEEPROM` will not be optimized out with `-g`.

Set a break at `init` and do a reset. When it breaks, press both buttons (I usually configure A and B to the same pin) and continue. EEPROM will be erased.

Reply:
I still don't quite get how adding this makes a difference? Do you set `debugResetEEPROM` to `TRUE` in the debugger, or how does it stop the condition from evaluating to false?

Anyhow, if this is left as it is, it will need at least a comment to explain why `debugResetEEPROM` should not be removed again in the next cleanup, as it looks unused.

As an alternative, should we simplify this even more, and not check `!isMPUSoftReset()` when compiled with `DEBUG` - I think the relaxation of 'only reset the config on power up' is appropriate for debug builds.